test/token/ERC20/extensions/ERC4626.t.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+import "erc4626-tests/ERC4626.test.sol";
+
+import {ERC20Mock} from "../../../../contracts/mocks/ERC20Mock.sol";
+import {ERC4626Mock, IERC20Metadata} from "../../../../contracts/mocks/ERC4626Mock.sol";
+
+contract ERC4626StdTest is ERC4626Test {
+    function setUp() public override {
+        _underlying_ = address(new ERC20Mock("MockERC20", "MockERC20", address(this), 0));
```

Comment:
IMO this would be cleaner/simpler. We are minting 0 to address 0
There is no "ownership" here that justifies using `address(this)` 

```suggestion
        _underlying_ = address(new ERC20Mock("MockERC20", "MockERC20", address(0), 0));
```

Reply:
@Amxx this indeed failed because `_mint()` (inside the constructor) reverts for address 0.  Do you have another suggestion other than address 0 here?

Reply:
My bad, lets keep it that way then.